ReactWindows/ReactNative/Views/Image/ReactImageManager.cs

```diff
@@ -161,15 +161,53 @@ public void SetSource(Border view, JArray sources)
             }
         }
 
+        /// <summary>
+        /// Enum values correspond to positions of prop names in ReactPropGroup attribute
+        /// applied to <see cref="SetBorderRadius(Border, int, double)"/>
+        /// </summary>
+        private enum Radius
+        {
+            All,
+            TopLeft,
+            TopRight,
+            BottomLeft,
+            BottomRight,
+        }
+
         /// <summary>
         /// The border radius of the <see cref="ReactRootView"/>.
         /// </summary>
         /// <param name="view">The image view instance.</param>
+        /// <param name="index">The prop index.</param>
         /// <param name="radius">The border radius value.</param>
-        [ReactProp("borderRadius")]
-        public void SetBorderRadius(Border view, double radius)
+        [ReactPropGroup(
```

Comment:
modify WPF implementation to match as well. test prop in Playground and provide code+screenshot for both UWP and WPF.

Reply:
@matthargett I have another PR ready to be submitted which fixes a bug in borderRadius and which has WPF part updated too. Is it okay if you approve this PR and I will do the Playground screenshots and code as part pf the next PR?

Reply:
I'd prefer to see a contribution upstream for a `border*Radius` example in RNTester. Does not need to be done for this repo.

Reply:
ReactNative already has examples of border*Radius at https://github.com/facebook/react-native/edit/master/RNTester/js/BorderExample.js see styles border6, border9, border10, border14.

Reply:
Yes - but not for Image

Reply:
@rozele PR into ReactNative for boredr*Radius test for Image seems to be in process of landing [here](https://github.com/facebook/react-native/pull/20264).